trl/trainer/utils.py

```diff
@@ -135,7 +135,8 @@ def __init__(
                 "The pad_token_id and eos_token_id values of this tokenizer are identical. "
                 "If you are planning for multi-turn training, "
                 "it can result in the model continuously generating questions and answers without eos token. "
-                "To avoid this, set the pad_token_id to a different value."
+                "To avoid this, set the pad_token_id to a different value.",
+                UserWarning,
```

Comment:
Warnings should not indicate normal behavior.
Use the appropriate warning type.

This warning can still occur in normal behavior. Any idea?

Reply:
What about `logger.info`?